azure/src/main/java/org/apache/iceberg/azure/adlsv2/ADLSLocation.java

```diff
@@ -30,14 +30,21 @@
  *
  * <p>Locations follow a URI like structure to identify resources
  *
- * <pre>{@code abfs[s]://[<container>@]<storage account host>/<file path>}</pre>
+ * <pre>{@code abfs[s]://[<container>@]<storageAccount>.dfs.core.windows.net/<path>}</pre>
```

Comment:
I feel we should stick to the URI format consistent w/ Hadoop's definition (without assuming the domain)